src/foam/u2/wizard/agents/DeveloperModeAgent.js

```diff
@@ -26,8 +26,10 @@ foam.CLASS({
 
   methods: [
     async function execute() {
-      const usersJunctions = (await this.userCapabilityJunctionDAO.select()).array;
-      const developerCapability = usersJunctions.find(j => j.targetId === 'developer');
+      const e = foam.mlang.Expressions.create();
+      const developerCapability = await this.userCapabilityJunctionDAO.find(
```

Comment:
definitely this is better - but also wonder why `this.crunchService.getJunction(ctrl.__subContext__, 'developer')` isn't used